package.json

```
@@ -44,5 +46,8 @@
       "dist"
     ],
     "preset": "ts-jest"
+  },
+  "np": {
+    "yarn": false
```

Not sure if you want to be explicit about the branch too to enforce only `main` being used for publishing?

```suggestion
    "yarn": false,
    "branch": "main"
```

---

`main` or `master` are the defaults